frontend/src/pages/Ingredient/Category.tsx

```diff
@@ -21,7 +21,7 @@ export default function Category() {
     return(
         <div>
             {category &&
-            category.drinks.map(singleCategory => <div onClick={() => nav(`/details=${singleCategory.idDrink}`)}>
+            category.drinks.map(singleCategory => <div key={Math.random()} onClick={() => nav(`/details=${singleCategory.idDrink}`)}>
```

Comment:
The key should not be random. React needs to identify the elements on the next render to update the correct ones. That won't work if they get a new random ID on each render.